beets/ui/__init__.py

```diff
@@ -924,7 +924,8 @@ def _raw_main(args, lib=None):
     # Special case for the `config --edit` command: bypass _setup so
     # that an invalid configuration does not prevent the editor from
     # starting.
-    if subargs[0] == 'config' and ('-e' in subargs or '--edit' in subargs):
+    if subargs and subargs[0] == 'config' \
+       and ('-e' in subargs or '--edit' in subargs):
```

Comment:
Isn't the indentation off by one? Eh. I guess flake8 didn't complain so it's probably all good. Just seemed odd to me.